profile/README.md

```diff
@@ -13,12 +13,12 @@ The Project is overseen by the [CDSC Technical Steering Committee (TSC)](https:/
 - Determines the allocation of resources (financial requests from LF Energy) will require a Strategic Member sponsoring the request which goes to the LF Energy Governing Board 
 - Participation in the CDSC TSC is open all, but voting is restricted to LF Energy General and Strategic members only.
 
-The Project additionally has a [Strategy Committee](https://github.com/carbon-data-specification/Strategy_Committee) which is focused on:
+The Project additionally has a [Strategy Committee](https://drive.google.com/drive/folders/17GynAWrlZ9IFXlOGhKZWOb852yhlV9dL?usp=sharing) which is focused on:
```

Comment:
@halliecramer1 I know you mentioned something about using Google docs vs. the mailing list files vs. github's wiki for document management. Is this the folder where you'll be keeping stuff? How can we ensure this folder sticks around for years (so people coming in years from now can look back through things if needed)?

Reply:
Yes, this is the folder where we will be keeping materials. We can create a policy where anything in the folder is more for coordination / process transparency rather than deliverables for the specification - if we produce a deliverable for the specification, we can commit to adding it to the Github instead

Reply:
ok, cool, thanks!